src/quo2/components/community/icon.cljs

```diff
@@ -0,0 +1,11 @@
+(ns quo2.components.community.icon
+  (:require [react-native.core :as rn]))
+
+(defn community-icon [{:keys [images]} size]
+  (let [thumbnail-image (get-in images [:thumbnail :uri])]
+    [rn/image {:source {:uri thumbnail-image}
+               :style  {:border-radius 50
+                        :border-width 0
+                        :border-color :transparent
+                        :width size
+                        :height size}}]))
```

Comment:
Yeah  I agree that we need to agree, but I don't think this is something I (or any other dev) should have to even think about. `make lint-fix` should address any of these formatting issues and any errors we find in p.r's should be handled by this tool. In the case there's some formatting we don't like the tool should be updated.

Perhaps I am being too optimistic and this is not possible in our current tech stack?? 🤔 

@yqrashawn - the auto formatter you are working on can handle this case?

Reply:
This kind of setting is usually set in editor. There's https://editorconfig.org/ for this, but generally user need to install an editor-config plugin for that.

VS Code https://stackoverflow.com/a/54270768/5671288
IntelliJ https://stackoverflow.com/a/16761228/5671288
seems Emacs and vim does this by default

Reply:
yeah, I agree. Imo we are just adding noise to our p.r's with formatting issues, when we could automate this process. if we can not automate this with `make lint-fix` then at the very least `make lint` should catch these issues and it is up to the dev to "manually" fix these issues. 
i.e the link check step on ci jobs will fail and this will notify the dev. Reviews can then focus on improving the quality of the code.

Is it just a case of we're not quite there yet, or is there something I am missing here?

Reply:
you mean, Hence, relying only on `make lint-fix` .. ?

Reply:
Maybe my comment wasn't clear enough. We will always rely on `make lint`, but as the last step to check. Relying on `make lint-fix` would be suboptimal during development, unless you (or any other dev) prefer that way. I never want to use `make lint-fix` in my daily development, I want my editor to automatically format the file I'm currently editing.

`make lint-fix` is slow compared to instant updates provided by IDEs/editors and you'll have to run it either on every commit or generate separate "fix lint" commits, but of course, it's up to you. Solutions with commit hooks fall in the same category, they make my interactions with git slow and I would disable them during development.

> yeah, I agree. Imo we are just adding noise to our p.r's with formatting issues

> Is it just a case of we're not quite there yet, or is there something I am missing here?

Unfortunately getting the formatter correct and capable of handling our own preferences (Status contributor preferences) is far from trivial and @yqrashawn had to actually report various issues to the maintainer of the tool. @yqrashawn just now opened a PR with new changes to introduce zprint o/